dev/devicelab/lib/framework/ab.dart

```diff
@@ -30,6 +32,48 @@ class ABTest {
     _addResult(result, _bResults);
   }
 
+  static void accumulateLengths(List<int> lengths, List<String> results) {
+    for (int i = 0; i < lengths.length; i++) {
+      if (results[i] != null) {
+        final int len = results[i].length;
+        if (lengths[i] < len) {
+          lengths[i] = len;
+        }
```

Comment:
Nit: line 38-41 can be simplified to `lengths[i] = math.max(lengths[i], results[i].length)`. (See https://api.dart.dev/stable/2.7.2/dart-math/max.html)